src/tex/crunch_functions.cpp

```diff
@@ -0,0 +1,110 @@
+#include <BS_thread_pool.hpp>
```

Comment:
Unused include?

Reply:
It was purely for the concurrency function, changed to `#include <thread>`.